### PR TITLE
Implemented music instrumentals website 

### PR DESCRIPTION
**Pull request by collinsmezie**
Music sound instrumentals website complete with Home and About page
links to the loom video here: https://www.loom.com/share/e188ff3d9e7a4a528ffa6a0aba447491

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,36 @@
-# Musical-instrumentals-website
+## Musical-instrumentals-website
 Jump on the Beat
+
+Desktop and Mobile version of Project Musical-instrumentals-website:
```
Comment (ShoiraTa) [inline README.md:4]: - [ ] Please make sure that a link to the deployed project is present in the README. You should use Github pages to deploy the project.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,36 @@
...
+
+### Site published at 
+- Coming soon...
+
+## Authors
+👤 **collins chime **
+
+- GitHub: [@collinsmezie](https://github.com/collinsmezie)
+- Twitter: [@collinsmezie1] collins mezie
+- LinkedIn: [collins mezie]
```
Comment (ShoiraTa) [inline README.md:38]: - [ ] Please make sure that the README contains an "Acknowledgements" section or similar that gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/)

File: index.html [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+            <p class="company">Record Label & Music streaming</p>
+            <h1 id="main-heading">STRAY BEATZ</h1>
+            <p class="introduction black">Lorem ipsum dolor sit, amet consectetur adipisicing elit. Dolorem adipisci, 
+               neque quisquam fugit officiis voluptates magnam amet ea ex molestiae quibusdam omnis
+            </p>
+    
+            <h2 class="time black">2022.10.15(THU) ~ 16(FRI)</h2>
+            <h3 class="venue black">@ Lekki phase 2, Lagos Nigeria</h3>
+        </div>    
+        <section id="header-about-section">
```
Comment (ShoiraTa) [inline index.html:56]: - [ ] The footer in about section should have the logo container and should have a white background for mobile view: 

Your design  | Behance design
------------- | -------------
Screenshot 2022-03-11 at 18 32 25  | Screenshot 2022-03-11 at 18 32 46

File: index.html [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+        </div>
+        <div class="see-more"><p>SEE ALL EVENT</p></div>
+    </div>
+    <section id="bodi-about-section"> 
+     <div class="about-bodi"> 
+        <div>
+            <div class="summit">
+                <h2>Creative commons global summit 2022 logo</h2>
+                <hr>
+            </div>
```
Comment (ShoiraTa) [inline index.html:112]: - [ ] OPTIONAL: Please consider centring the logo in `About` page (Mobile view): 

Screenshot 2022-03-11 at 18 34 08

File: index.html [hunk 5]
```diff
@@ -0,0 +1,177 @@
...
+                <div><h3 class="red">Play</h3></div>
+                <div><p>Listen to the speakers from various countries about the messages of sharing and opening</p></div>
+            </div>
+            <div class="song">
+                <div><img src="media/music.svg" alt="Disk" width="60" height="60"></div>
+                <div><h3 class="red">Play</h3></div>
+                <div><p>Listen to the speakers from various countries about the messages of sharing and opening</p></div>
+            </div>
+            <button id="selectsong" type="button">Jump on the Beat</button>
+        </div>
```
Comment (ShoiraTa) [inline index.html:103]: - [ ] Please center the button :
Screenshot 2022-03-11 at 18 35 59
You can simply do that by using flexbox `align-items` property:

```
#selectsong {
    margin: 40px 0 40px 0;
}
.songs-list {
   align-items: center;
}
```

File: styles.css [hunk 6]
```diff
@@ -0,0 +1,734 @@
...
+  background: url("media/background.jpg") no-repeat center center fixed;
+  -webkit-background-size: cover;
+  -moz-background-size: cover;
+  -o-background-size: cover;
+  background-size: cover;
+}
+
+@media (max-width: 768px) {
+  .window {
+    display: block;
```
Comment (ShoiraTa) [inline styles.css:64]: - [ ] In your project you are using desktop first approach to style, but according to requirements  you should use mobile-first approach: 

Screenshot 2022-03-11 at 18 42 30

This is how media queries are written for mobile-first approach : 

Screenshot 2022-03-11 at 18 43 40

**Review by ShoiraTa — CHANGES_REQUESTED** [review]
Hi @collinsmezie ,

:thumbsup: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

There are some issues that you still need to work on to go to the next project but you are almost there!



 ## Required Changes  :recycle: 

_Check the comments under the review._ 

***
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear by mentioning my Github username.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

* * *
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (collinsmezie) [reply index.html:56]: The page has to be refreshed when viewed in developer mode(mobile view) in order to render correctly.
Screenshot from 2022-03-11 19-33-14

Comment (ShoiraTa) [reply index.html:56]: I can see that it works when refreshed🙌

**Review by Lordkaito — APPROVED** [review]
Hi @collinsmezie, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**